src/index.js

```diff
@@ -27,6 +29,7 @@ const home = () => {
       <PokeImage />
       <NameInput />
       <PokemonTypes />
+      <PokemonVarietySelect index=${0} />
```

Comment:
if this should be a default value, maybe set that in the component itself, 
```
export default ({index = 0}) => {
```

Reply:
this is part of the assumption that each pokemon will (soon) have an index (and you'll have pokemon set of 0 through 5). It should not be a default value.

Reply:
ah, index is probably a bad variable name then... don't worry about this for now